pyscript.core/esm/worker/class.js

```diff
@@ -28,8 +29,15 @@ export default (...args) =>
         const bootstrap = fetch(url)
             .then(getText)
             .then((code) => postMessage.call(worker, { options, code }));
-        return defineProperty(worker, "postMessage", {
-            value: (data, ...rest) =>
-                bootstrap.then(() => postMessage.call(worker, data, ...rest)),
+        return defineProperties(worker, {
+            postMessage: {
+                value: (data, ...rest) =>
+                    bootstrap.then(() =>
+                        postMessage.call(worker, data, ...rest),
+                    ),
```

Comment:
Are these properties intentionally non-writeable non-configurable non-enumerable? If so, why?

Also, a dumb simplification:
```suggestion
                value: (...args) =>
                    bootstrap.then(() =>
                        postMessage.call(worker, ...args),
                    ),
```

Reply:
that dumb simplfication breaks expectations on `postMessage` length which is `1` ... see `postMessage.length` in your console.

Beside that, in which scenario you think changing that is helpful / needed?

On top of that, the original `postMessage` is trapped once and nothing can change it, so this reflects the fact the `xworker` is sealed and I think that's fine.

Reply:
P.S. `postMessage.call(worker, ...args),` can also be simplified as `postMessage.apply(worker, args),` ;-) but whenever there is an arity to preserve I try to do so and since dropping the arity brings zero benefits I think the way it is now it's fine, still curious to understand why you'd like to have that crucial utility overwritable or re-configurable.

Reply:
> why you'd like to have that crucial utility overwritable

Well you're overriding `postMessage` right here. If this "crucial utility" weren't overridable you wouldnt be able to do this. Also if you didn't need to override it, you wouldn't have bothered to seal it. So it seems like a weird thing to do to me.

Reply:
I am overriding my own API, the sealed override is an implementation detail nobody should (and would) care about? It's not that I am changing Worker.prototype.postMessage, it's the API offered that is just trusting it works as it's meant to.